packages/devtools/lib/src/timeline/cpu_profile_protocol.dart

```diff
@@ -36,9 +36,16 @@ class CpuProfileData {
 
   void _processStackFrames(Response response) {
     stackFramesJson.forEach((k, v) {
-      final stackFrame = CpuStackFrame(k, v['name'], v['category']);
-      final parent = stackFrames[v['parent']];
-      _processStackFrame(stackFrame, parent);
+      final String stackFrameName = v['name'];
+
+      // Do not process [Native] events. They do not provide any helpful
```

Comment:
nit: make these a flag. Developers trying to use this tool to debug dart embedding issues may feel differently.

Reply:
Added a todo to add a checkbox. Flagged to exclude Native events by default right now.